### PR TITLE
Testing for text-rotate and text-max-width

### PR DESCRIPTION
**Pull request by johngschulz**
Testing for text-rotate and text-max-width
Cleaned up testing, removed unused test file.

### CODE REVIEW

File: modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/SymbolMBLayerTest.java [hunk 1]
```diff
@@ -31,17 +36,20 @@
     SymbolMBLayer testLayerDefault;
     SymbolMBLayer testLayer;
     MBStyle defaultStyle;
-
+    MBStyle style;
+    List<FeatureTypeStyle> ftsd;
+    List<FeatureTypeStyle> fts;
```
Comment (tbarsballe) [inline modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/SymbolMBLayerTest.java]: These variable names are a bit ambiguous, especially considering the types are identical

**Comment by johngschulz** [comment]
Updated FeatureTypeStyle naming and squashed commits.

**Comment by tbarsballe** [comment]
Thanks for squashing the commits. I'll wait for the CI build to complete, then merge.

**Comment by tbarsballe** [comment]
Looks like we have some conflicts here again, probably from me merging #1621

**Comment by johngschulz** [comment]
Conflicts re-fixed

**Comment by tbarsballe** [comment]
Thanks - looks like your squash has overwritten the original commit message. Can you fix that, then I can merge.

**Comment by johngschulz** [comment]
Commit message amended.

**Comment by tbarsballe** [comment]
Each time I go to merge this, it gets stomped by another of your PRs I recently merged - we have merge conflicts again...

**Pull request merged**